ironfish-wasm/src/lib.rs

```diff
@@ -59,17 +59,17 @@ pub fn create_key_to_js() -> Key {
 }
 
 #[wasm_bindgen(catch, js_name = "generateNewPublicAddress")]
-pub fn create_new_public_key_to_js(private_key: &str) -> Key {
-    console_error_panic_hook::set_once();
+pub fn create_new_public_key_to_js(private_key: &str) -> Result<Key, JsValue> {
```

Comment:
should JsValue be `WasmSaplingKeyError`?

Reply:
Looks like wasm-bindgen may not support this, even though WasmSaplingKeyError implements From<JsValue>